### DISCUSSION TITLE
Bio energy subsidies, Tau and FPI

### DISCUSSION THREAD

**Discussion by jansteinhauser** [magpiemodel/magpie#307] [Show and tell]
Hi everyone,

We are currently working on creating a lookup table to feed our energy model with bio energy potentials based on different combinations of bio energy and pollution prices put into MAgPIE ([comparable to this GLOBIOM lookup table](https://github.com/iiasa/GLOBIOM-G4M_LookupTable#lookup-table-structure)). I’d like to share the first few outputs with you and get your input on how to proceed, as we're getting rather high potentials combined with very high Tau and food prices.

My core changes so far are simple:
a)	Linearly implement the target BE price (through `c60_bioenergy_subsidy`) between 2020 and 2100

`if(m_year(t) <= 2020,` 
`i60_bioenergy_subsidy(t) = 0;`
    
`elseif (m_year(t) > 2020) and (m_year(t) < 2100),`
` i60_bioenergy_subsidy(t) = c60_bioenergy_subsidy/(2100-2020) * (m_year(t) - 2020)`

`else i60_bioenergy_subsidy(t) = c60_bioenergy_subsidy);`

b)	Apply the subsidy to all BE carriers based on each carrier’s energy content
`q60_bioenergy_incentive(i2).. vm_bioenergy_utility(i2) =e=` 
`sum((k1st2ndres,ct), vm_dem_bioen(i2,k1st2ndres)* fm_attributes("ge",k1st2ndres) * (-i60_bioenergy_subsidy(ct)));`

Where `k1st2ndres(kall)` is the set containing all types of bioenergy carriers: 
`/betr, begr, oils, ethanol, res_cereals, res_fibrous, res_nonfibrous/`

In the following, I compare four runs: Two with these code changes and (relatively strong) incentives of 30 and 60 US$00 / GJ, called `210818_030` and `210818_60`, respectively. And two runs using the default settings with the exception of the subsidy, which is 600 and 1,200 USD05 / tDM (as rough equivalents to the 30 and 60 USD / GJ at, on average, 20 GJ / tDM), called `210818_default_30` and `210818_default_60`, respectively.

With the rather strong incentive of 1,200 US$05 / tDM, MAgPIE gives a bio energy demand of roughly 40,000 MtDM in 2100. For comparison, most runs in the EMF Database put bio energy total between 100 and 300 EJ by 2100, with some runs going as low and high as 50 and 600 EJ. Similarly, for energy crop production, most runs sit between 2,000 and 15,000 MtDM/year, with a few going as high as 40,000 MtDM - which puts this run as high as these somewhat extreme upper outliers.  For 600 USD, it’s about 15,000 Mt DM – far less, but still above most EMF runs. Also, due to the default focus of the subsidy, this is mostly 1st generation while most of the DM in the database is 2nd generation.

Using the alternative realization, bio energy demand goes as high as roughly 350,000 and 150,000 MtDM per year, respectively, with a much stronger focus on 2nd generation and residues though. In EJ, this puts the 60 USD00 run at around 6,300 EJ per year by 2100 and the 30 USD00 run at around 2,800 EJ per year. 

![Demand_Bioenergy](https://user-images.githubusercontent.com/72768733/130372119-f270d261-6f69-40a4-bbd9-1fbbe0eec637.png)

![Demand_Bioenergy_EJ](https://user-images.githubusercontent.com/72768733/130408385-c641f04b-6f79-4339-beae-4fc9b277d62b.png)

So, in default and even more so in the new realization the bio energy potential seems to be quite excessive, obviously affecting other variables. While, depending on the PA and LUC policies selected, this is in parts enabled by shifts towards cropland and pasture, land-use intensity seems to be at least equally as important.

![Productivity_Landuse_Intensity_Indicator_Tau](https://user-images.githubusercontent.com/72768733/130417206-7cc6dd8d-9a93-47e3-8001-619dfd2db169.png)
 
And, as a side note, the food price index is going through the roof.
![Prices_Food_Price_Index](https://user-images.githubusercontent.com/72768733/130417688-851648ac-c13f-4d2e-8902-8e7523decae6.png)

So, the FPI ends up on a level that doesn't seem to be very sustainable. And we're wondering how realistic such a Tau is and what a sensible constraint for both of these aspects could be. Have you had similar results in the past and how would you go about this? Is there maybe something I've missed in the equations?

And, as a side issue, for high subsidies some PA/LUC combinations seem to cut off way before 2100. Demand simply suddenly drops to 0. Any idea why this might happen? It fits this topic, but I may open an extra discussion if there's need for that.

Thanks!
Jan

**Comment by tscheypidi** [comment]
Hi Jan,

interesting results you have there :) 

I don't recall having seen any scenario with such extreme increases in tau (we are talking about 12x productivity increase globally!), but the overall behavior (extreme productivity and food price increases) is something we see on a regular basis. It happens if the model get an infinite flow on income for an activity which is out-competing all other activities in terms of profitability. 
In this particular case that bioenergy subsidy seems to be so high that it makes sense from an economic perspective to extremely intensify production to get as much bioenergy produced as possible, ignoring all negative side effects which come with it. Did you check what happens in terms of cropland expansion? I could even imagine that the bioenergy subsidy might overcompensate the emission taxation costs associated with deforestation which could lead to strong deforestation in order to produce more bioenergy. 

In reality such a scenario would not happen as these increases in food prices would be unacceptable and probably nobody would be willing to subsidize bioenergy production to that amount without limiting the total quantity. 

Possible solutions could be for the emulator calculation to throw out scenarios in which the food price index surpasses a pre-defined threshold (reflecting what would be assumed socially acceptable / sustainable), or thinking about other constraints describing which scenarios would be assumed feasible/accetable. I would not limit tau, as there it is really hard to make a case why certain levels cannot be surpassed. From my perspective it is less about the question what productivity levels can be reached but more about the question what other constraints must be fullfilled (such as a affordable food price level or maximum amount of capital that can be spent on productivity increases and production). 

All that said, I am still surprassed by that extreme increases in tau you are seeing. We typical see global tau levels in the range of 5-8 in these scenarios in which a certain activity is over-incentivized, so this scenario is conserably more extreme than what I am normally used to. It might make sense to crosscheck all conversion factors again in order to see whether the numbers are alright.
